app/views/foreman_ansible/ansible_roles/_included_roles.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="form-group">
```

Comment:
I see what you tried to do here, but we follow a more intuitive model (e.g: user would wonder why do these roles not show on 'Selected items' on other inherited stuff.

![screenshot from 2016-08-12 13-08-15](https://cloud.githubusercontent.com/assets/598891/17620617/d9dafcaa-608d-11e6-91ce-832dabee4045.png)

to look like this (example from nested taxonomies):

![screenshot from 2016-08-12 13-12-28](https://cloud.githubusercontent.com/assets/598891/17620727/96e804be-608e-11e6-869a-d4d0fd5613bb.png)